src/Writers/HTMLWriter.jl

```diff
@@ -891,13 +891,13 @@ end
 analytics_script(tracking_id::AbstractString) =
     isempty(tracking_id) ? Tag(Symbol("#RAW#"))("") : Tag(:script)(
```

Comment:
`Tag(:script)` tag may be redundant with `<script>`

Reply:
Sorry for the delay. But yes, you're right, this would be redundant. In fact, you'd end up with nested `<script>` tags, which means that I don't think this would work at all at the moment.

Instead, I'd recommend explicitly constructing both of the `<script>` tags in Julia. It should be fine if `analytics_script` returns an array of `Tag`s.

Reply:
Is the syntax for constructing tags in Julia documented? (e.g., how do you do `<script async src="...">`?)

Reply:
Yep: https://juliadocs.github.io/Documenter.jl/stable/lib/internals/dom/

Also, heaps of usage examples in `HTMLWrite.jl`.